packages/core/src/_common/base/socket_base.js

```diff
@@ -513,10 +513,10 @@ const proxied_socket_base = delegateToObject(BinarySocketBase, () => BinarySocke
 const proxyForAuthorize = obj =>
     new Proxy(obj, {
         get(target, field) {
-            if (typeof target[field] !== 'function') {
+            if (typeof target[field] !== 'function' && undefined) {
```

Comment:
```suggestion
            if (typeof target[field] !== 'function') {
```
@yaswanth-deriv, why did you add the `undefined` here? 🤔